solutions/client-landing-zone/client-folder/standard/applications-infrastructure/host-project/org-policies/exceptions/compute-restrict-cloud-nat-usage-except-host-project.yaml

```diff
@@ -0,0 +1,44 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#########
+#
+# GCP Organization Policies
+# Org policies that correspond with a Guardrail will contain a label indicating what Guardrails it helps in enforcing
+# https://cloud.google.com/resource-manager/docs/organization-policy/org-policy-constraints
+#
+# Constraint: constraints/compute.restrictCloudNATUsage
+#
+# This list constraint defines the set of subnetworks that are allowed to use Cloud NAT.
+#
+# This exception is for the host project as it requires use of Cloud NAT.
+#
+#########
+apiVersion: resourcemanager.cnrm.cloud.google.com/v1beta1
+kind: ResourceManagerPolicy
+metadata: # kpt-merge: policies/compute-restrict-cloud-nat-usage-except-host-project-id
```

Comment:
all these `# kpt-merge` comments and kpt annotations should be removed to minimize future package update issues...
this snippet (from push.sh in the tools repo) can be run to cleanup all the files:
```bash
    find . -type f -name "*.yaml" | while read -r file; do
      # remove node "cnrm.cloud.google.com/blueprint"
      yq eval 'del(.metadata.annotations."cnrm.cloud.google.com/blueprint")' -i "$file"
      # remove node "internal.kpt.dev/upstream-identifier"
      yq eval 'del(.metadata.annotations."internal.kpt.dev/upstream-identifier")' -i "$file"
      # remove empty annotation node
      yq eval 'del(.metadata.annotations | select(length==0))' -i "$file"
      # remove strings  " # kpt-merge: .*"
      sed -i 's/ # kpt-merge: .*//' "$file"
    done
```

Reply:
Thanks. Fixed.